### PR TITLE
Pull it all

### PR DESCRIPTION
**Pull request by xMooz**
racist

### CODE REVIEW

**Comment by staticinteger** [comment]
Not sure how I feel about this. It will be interesting to see what this turns into.

**Comment by orionblastar** [comment]
I think we are seeing the social justice warriors and liberal progressives try to insert ineffective left-wing propaganda into Linux. Either that or the Feminist Software Foundation is some troll account subject to Poe's law.

If you cannot Pull Linux, try Forking Linux. Apologies to Adria Richards who doesn't get the joke.

**Comment by thelostt** [comment]
What the hell is that?

**Comment by rumpelsepp** [comment]
No idea, but it's funny. :D

**Comment by Jheengut** [comment]
On Sunday, January 18, 2015 06:49:32 AM Stefan Tatschner wrote:
> No idea, but it's funny. :D
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/torvalds/linux/pull/156#issuecomment-70410819
The joke of the year

**Comment by rumpelsepp** [comment]
maybe

File: README.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+4. [Choose portability over efficiency.](http://en.wikipedia.org/wiki/Deadbeat_parent)
+5. [Store data in flat text files.](http://thinkprogress.org/health/2014/06/16/3449302/media-women-portrayals/)
+6. [Use software leverage to your advantage.](http://host.jibc.ca/seytoolkit/what.htm)
+7. [Use shell scripts to increase leverage and portability.](http://en.wikipedia.org/wiki/Dictatorship)
+8. [Avoid captive user interfaces.](http://en.wikipedia.org/wiki/Slavery)
+9. [Make every program a filter.](https://en.wikipedia.org/wiki/Gender_role)
+
+To eradicate the inherent root of Toxic Bigotry in the UNIX Philosophy, the ToleranUX Philosophy is proposed as such:
+
+1. **Large is beautiful.**  Small programs benefit from small privilege, but are no better than large programs.  We need more large programs to make up for this.  **Unused RAM is wasted RAM** — if a single `ls` doesn't use up at least 50% of installed RAM then it is condoning the toxic culture of fat shaming.
```
Comment (xbony2) [inline README.md]: This line is basically proof this is a joke :P

**Comment by master801** [comment]
This pull request is fuckin' stupid is a terrible attempt to troll someone.

**Comment by orionblastar** [comment]
This is some sort of stupid politics publicity stunt:
http://arstechnica.com/information-technology/2015/01/toleranux-satirical-linux-fork-mocks-calls-for-open-source-diversity/

It only modifies the documentation and source code comments to mock Linus for some speech he gave recently about diversity in the FOSS community. 

http://www.reddit.com/r/TumblrInAction/comments/2sqjat/twitter_and_tumblr_feminists_call_for_the_removal/

This is Twiter and Tumblr Feminists protesting Linus and want him removed because he is a white CIS male. It is called tearing down the white straight male because they dominate at STEM.

**Comment by j-siddle** [comment]
I'm pretty sure it's supposed to be a satirical argument against perceived
feminist attacks against white/male dominated culture in IT, etc. It's a
straw man argument though, better to just ignore it and apply some rational
thinking than to get caught up in a non-existent battle.



On 19 January 2015 at 04:15, orionblastar <notifications@github.com> wrote:

> This is some sort of stupid politics publicity stunt:
>
> http://arstechnica.com/information-technology/2015/01/toleranux-satirical-linux-fork-mocks-calls-for-open-source-diversity/
>
> It only modifies the documentation and source code comments to mock Linus
> for some speech he gave recently about diversity in the FOSS community.
>
>
> http://www.reddit.com/r/TumblrInAction/comments/2sqjat/twitter_and_tumblr_feminists_call_for_the_removal/
>
> This is Twiter and Tumblr Feminists protesting Linus and want him removed
> because he is a white CIS male. It is called tearing down the white
> straight male because they dominate at STEM.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/torvalds/linux/pull/156#issuecomment-70444623>.
>



-- 
email: jim@jamessiddle.net
web: www.jamessiddle.net

"There's an old saying in Tennessee -- I know it's in Texas, probably in
Tennessee -- that says, fool me once, shame on --shame on you. Fool me --
you can't get fooled again." - George W. Bush

**Comment by jakesyl** [comment]
Feminazis

File: README.md [hunk 2]
```diff
@@ -0,0 +1,402 @@
+
+  ![FSF](http://i.imgur.com/ZhTU8r3.png)
+  
+  ![ToleranUX Shell Prompt](http://i.imgur.com/ZVjbJeS.png)
+
+***
+
+A project of the Feminist Software Foundation. Feminist software is a cornerstone of any modern free society. We build this foundation.
+
+> *"A Linux without Linus Torvalds is a better community. A more independent and self-determinate community. A more welcoming community." — @shanley on Twitter*
```
Comment (jakesyl) [inline README.md]: @torvalds, awaiting your opinion on this

File: README.md [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+2. **Allow each program to do whatever it chooses to.**  The tyranny of the user will not stand.
+3. Rushing a prototype privileges the traditional masculine trait of focusing on a single issue; instead, the planning and mockup stage should be paramount to address the visual and spatial draws of feminine coding.  **The modern age is a UX age, and designers should be paid more than coders.**
+4. **Choose inclusiveness over meritocracy.**  The judgement of merit is entirely subjective, so the ideal software project (which **ToleranUX** aspires to be one) should be inclusive to all commits and contributors.
+5. **Store data in text files which implement a curvy-brackets syntax**; the fetishisation of "flat files" betrays the inherent unrealistic sexual fantasy that [male brogrammers](http://en.wikipedia.org/wiki/Brogrammer) hold.
+6. Software should never be "used" to anybody's "advantage"; instead, **the consent of software must be asked first**.  Suggesting that any program should ever be "used" is the epitome of creepiness.
```
Comment (jakesyl) [inline README.md]: Software Rights!

**Comment by jakesyl** [comment]
This is honestly the best troll ever

**Comment by strwbrry** [comment]
Wasn't sure if joke or serious. Honestly SJW are not far off this.

https://github.com/bmuller/gender_detector/pull/14

**Comment by aandis** [comment]
lol. Let's see what @torvalds has to say on this,

**Comment by jakesyl** [comment]
haha the @torvalds response will be legendary

**Comment by RobertWHurst** [comment]
If he even bothers. I kind of feel like he's likely to ignore this PR, assuming he checks GitHub at all.

**Comment by strwbrry** [comment]
Some guy was fired for not merging a PR where some girl removed male pronouns and replaced them with gender-neutral ones, he might ignore this to avoid a backlash, not that he has ever shown signs of ever giving a fuck.

**Comment by jakesyl** [comment]
Yay! let's put your name on the contributors list for getting rid of some male pronouns!

**Comment by RobertWHurst** [comment]
Just be careful how you guys respond to this PR. I does look satirical, but all joking aside, this thread may turn in to a quote mine.

**Comment by jakesyl** [comment]
Continuing with logical comments:
```
Absolutely no coding experience is necessary: all code are equal in the eyes of the Feminist Software Foundation. There is no objective way to determine whether one person's code is better than another's. In light of this fact, all submitted code will be equally accepted. However, marginalized groups, such as wom*n and trans* will be given priority in order to make up for past discrimination. Simply submit a pull request for any submission, whether code, artwork, or even irrelevant bits — nothing is irrelevant in the grand struggle for a Truly Tolerant UNIX-ike Kernel!:
```

Sexism occurs in this PR here:

```
However, marginalized groups, such as wom*n and trans* will be given priority in order to make up for past discrimination
```

**Comment by jakesyl** [comment]
Other satires: https://gitorious.org/c-plus-equality/c-plus-equality/source/cefcfb4276889bd3833c539225a9cdfd3eb16d33

**Comment by orionblastar** [comment]
First off I believe that diversity is strength. I welcome everyone into IT and programming. This is not the way to get equality by sabotaging someone else's work with a political agenda. 

I encourage people of all gender, race, sexual orientation, national origin, disability, religion or non-religion, whatever to visit this website: http://noexcuselist.com/ and find an area they want to learn in and learn it and then learn how to code and contribute in a positive way to Github.

Satire, mockery, sarcasm, does not help it only hurts the developers who work hard on these projects.

**Comment by joshumax** [comment]
...I can't wait until Linus looks at this PR...

*meanwhile, at The Linux Foundation*

Linus: "Looks like it's that time of day again to push linux-next changes to my GitHub mirro--hey, looks like a lot of people have been pinging me, I wonder why they'r--SON OF A--"

__Two Hours Later__

**Comment by jakesyl** [comment]
@orionblastar thanks for no excuse list!  Also sometimes satire is an effective way to get your point across... also @joshumax that's a really old 404 haha

**Comment by ariccio** [comment]
# REALLY? THESE GUYS AGAIN??!???

It's pathetic. Not funny.

[Please. You were deleted the first time.](https://github.com/torvalds/linux/pull/82)

**Comment by noflag** [comment]
yes gators, we get it, you made a funny fake feminist linux thing. maybe go outside, have a jog, read a book? honestly it's getting pathetic. sorry someone made fun of your toys :-(

**Comment by CombustibleLemon** [comment]
@modwizcode

( ͡° ͜ʖ ͡°)

**Comment by ChrisMCMine** [comment]
100% 4chan at its best

**Comment by saatanaperkele** [comment]
>Gators
what does this even mean? I don't like American football.

**Comment by noflag** [comment]
Gamergaters. They think it's some kind of funny parody of feminists

On February 11, 2015 1:16:10 PM PST, Uncle Benis <notifications@github.com> wrote:
>>Gators
>what does this even mean? I don't like American football.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/torvalds/linux/pull/156#issuecomment-73965976

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by saatanaperkele** [comment]
Oh, that stuff. I don't even like that movement and I think it's a good parody of the radical left. My mother was actually a huge feminist in Finland years ago and she thinks a lot of this is hillariously insane.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+### News and TODO
+
+The specification is almost done.  The parts where there is input still needed are: supported architectures and form factors; inclusion of more `coreutils` to be corrected to feminist thought; and inclusion of more Kernel Daemons under ~~SystemD~~ `SystemV`.
+
+An initial implementation of the `TolernUX-coreutils` is now done in as a ~~`shell .rc`~~ `User-Interactive Free To Be Thee Devices (UIFTBTD) .rc`, which can be [found here](https://github.com/The-Feminist-Software-Foundation/ToleranUX/tree/mistress/ToleranUX-utils).
+
+### How to contribute
+
+Absolutely no coding experience is necessary: all code are equal in the eyes of the Feminist Software Foundation.  There is no objective way to determine whether one person's code is better than another's. In light of this fact, all submitted code will be equally accepted.  However, marginalized groups, such as wom\*n and trans\* will be given priority in order to make up for past discrimination.  Simply submit a pull request for any submission, whether code, artwork, or even irrelevant bits — nothing is irrelevant in the grand struggle for a Truly Tolerant UNIX-ike Kernel!
+
```
Comment (MichalPaszkiewicz) [inline README.md]: 'There is no objective way to determine whether one person's code is better than another's.'

Seriously?

Comment (Jaykay-x) [inline README.md]: kidding me?
2015年3月11日 18:22于 "Michal Paszkiewicz" <notifications@github.com>写道：

File: README.md [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+### `shell`
+
+To refer to the user interface as a `shell` is troubling in nature due to the fact that is an obstacle for those who are recently coming out from the psychoanalytical "shell" of trans\*sexuality or trans\*genderism. To refer to as a UI as a `shell` implies that all programs ran inside the `shell` is never going to leave the `shell`. As such, all `shell` processes are to be referred to as **`User-Interactive Free To Be Thee Devices (UIFTBTD)`**. To further this, all `UIFTBTD`s are given permission to process data and perform calculations anywhere inside the Computerspaces, not just the "Shell".
+
+### `kill`
+
+To `kill` a process is a triggering term to those who have been affected by death or killings in their life.  Instead of `kill`-ing a process, which is something immoral and a result of toxic masculinity, ToleranUX opts to `fire` processes.  Unlike `kill`-ing a process, `firing` a process allows it to still live, just no longer performs any function.  In addition to being able to manually `fire` any process that offends you, **ToleranUX** will automatically `fire` any process that is determined to be too offensive.  Likewise, `killall` is now `fireall`.
+
```
Comment (MichalPaszkiewicz) [inline README.md]: I'm assuming garbage collection - a process viewed as a masculine job by the patriarchy - will now also come to an end.

**Comment by jakesyl** [comment]
^^

**Comment by noflag** [comment]
Bullshit politics and internet drama shouldn't have this much interaction on a core linux project. This pull request is a joke by supporters of "gamergate" to mock feminists.

**Comment by Jheengut** [comment]
On Wednesday, January 21, 2015 07:35:25 AM Jake wrote:
> Yay! let's put your name on the contributors list for getting rid of
> some male pronouns!
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/torvalds/linux/pull/156#issuecomment-70858260
Maybe You don't realise but pritvi(or prithvi) is technically a female 
name but used mostly by the males

but prithviraj is a male name meaning earth ruler

**Comment by karansingh9008** [comment]
Pft.. Can't believe this thread is still going on.. 😕

**Comment by ariccio** [comment]
Y'all are still at this? Please spend your time on something else. There are tons of open source projects that are in desperate need of help.

**Comment by GBGamer** [comment]
Stop doing this. Seriously. You are filling up my inbox with your annoying bullshit.

**Comment by tucnak** [comment]
This is fun as fuck! Keep on doing this, Im lovin it!

**Comment by RobertWHurst** [comment]
@GBGamer on the sidebar to the top right of this thread there is a unsubscribe button. This will solve your spam issue.
screen shot 2015-08-20 at 1 58 12 pm

**Comment by cunidev** [comment]
It's pathetic to attract the attention of a serious and stable community on you with such a ridiculous pull request. You're even damaging the reputation of SERIOUS femisists! Stop it, it has absolutely no sense.

**Comment by phase** [comment]
So when is this getting merged?

**Comment by ankghost0912** [comment]
Just imagine the code:
#include<stdio.h>
#include<> //we must include everything. no one should feel left out
...
500 lines later
void xyz()
{
 //will do later. I don't feel that the functions name is cute enough.
}

main()
{
  printk("Hello kernel! How are you ! OMG I'm so excited, to talk to you. You're so cute and cuddly"!)
}

File: README.md [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+6. Software should never be "used" to anybody's "advantage"; instead, **the consent of software must be asked first**.  Suggesting that any program should ever be "used" is the epitome of creepiness.
+7. Use Shell Suggestions ("scripts" is forcing minorities to succumb to oppression and thus violates the safespace) to **increase equality and inclusiveness**.
+8. Avoid oppressive and difficult interfaces designed for white, cis-gendered males.  The keyboard is obsolete and so is [the CLI](http://en.wikipedia.org/wiki/Command-line_interface).  **A touchscreen interface is the future** — an interface which does not have the [ableist barriers](https://en.wikipedia.org/wiki/Ableism) against Persons of No Touch Typing Ability (PoNTTA).
+9. Filter programs according to the [**progressive stack**](https://en.wikipedia.org/wiki/Progressive_stack).
+
+Kernel Hacking Etiquette
+========================
+
+*Before you make a comment or suggestion, think about this:  
+Could someone conceivably take offence at what I'm saying? Am I going to hurt someone's feelings?
```
Comment (MartinDomig) [inline README.md]: I take offence at this.

File: README.md (hunk 2 above)
Comment (nkeck720) [inline README.md]: Wow. I might just go 'rm -rf' myself after that.

File: README.md (hunk 3 above)
Comment (nkeck720) [inline README.md]: This is a huge joke.

Comment (marctmiller) [inline README.md]: hah github made me redact the 'rm -rf yourself' commentary

File: README.md (hunk 4 above)
Comment (nkeck720) [inline README.md]: "No coding experience is necessary." Oh, so now we are submitting crappy code to the OS that, oh, 60% of the world's servers use?

**Comment by nkeck720** [comment]
Hmm, I wonder what @torvalds will say about this BS?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+  * [Supported form factors](#supported-form-factors)
+  * [Artwork](#artwork)
+  * [Coding Style](#coding-style)
+
+ToleranUX
+===============
+
+  ![ToleranUX logo](http://i.imgur.com/nDX7jA1.png)
+
+**ToleranUX** (like UNIX, but with more Toblerone and Tolerance) is the world's first [UNIX-like](https://en.wikipedia.org/wiki/Unix-like) operating system kernel that adheres to the 21st Century modern tenets of Equality, Inclusiveness, and Tolerance.  **ToleranUX** is created to revolutionise the [Toxic Meritocracy](http://geekfeminism.wikia.com/wiki/Meritocracy) that permeates the [FLOSS (Free, Libre, and Open Source Software)](http://geekfeminism.wikia.com/wiki/FLOSS) world that has proved itself to be the crux of divisiveness, the cause of the gender imbalance in IT, and the bane of True Equality.
```
Comment (nkeck720) [inline README.md]: Equality, eh? So all this hating on cis white men is equality?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+
+The filesystem can add the data, if it consents, or it may refuse. Data is not "read" from the file system, it is given. Data is not "written," it is offered.
+
+### IPC and Pipes
+
+The [traditional UNIX Pipes](https://en.wikipedia.org/wiki/Pipeline_(Unix)) are deemed offensive for their [phallic](https://en.wikipedia.org/wiki/Phallus) and therefore problematic tone.  Instead of the user [dictating](https://en.wikipedia.org/wiki/Dictatorship) the order of programs passing data between themselves, programs should be able to choose when and where to **organically congregate** amongst themselves, and pass either full or partial data between input and output.
+
+More advanced IPC is to be *guided*, never *dictated*.  The obsolete `D-Bus` mechanism is rewritten in Node.js and *accepted* into kernel-space for inclusion.  This IPC shall be named **"cervical canals"**, denoted by the symbol `Ω`, as a feminist respite of the Patriarchal construct **"UNIX Pipes"** and its `|`.
```
Comment (nkeck720) [inline README.md]: I thought this couldn't get any better and then I read this

File: README.md (hunk 5 above)
Comment (nkeck720) [inline README.md]: I'm done with this.

File: README.md (hunk 3 above)
Comment (phase) [inline README.md]: these rights are making me uncomfortable

Comment (marctmiller) [inline README.md]: just look at them as writes instead

Comment (nkeck720) [inline README.md]: Yeah....

File: README.md (hunk 2 above)
Comment (2003ryans) [inline README.md]: >feminists are trying to eliminate linus
k

**Comment by meskarune** [comment]
Guys, Feminist Software Foundation is a troll account, not an actual feminist group. Don't believe every damn thing you see on the internet. Report the account and move on.

**Comment by jakesyl** [comment]
Want to punish the trolls?  Say you'll merge it if they resolve all the merge conflicts....

**Comment by marctmiller** [comment]
linus doesn't merge pull requests.

**Comment by orionblastar** [comment]
They need a safe place on the Internet to hide out because they got their feelings hurt. 

We should make a code of conduct that says they cannot add political stuff to the repository. There are still conflicts.

Geez, some of the edits don't make any sense. I guess if these SJWS/Feminists study liberal arts instead of a STEM, they don't have the skills to understand why these edits conflict and got rejected.

**Comment by meskarune** [comment]
@orionblastar its a troll account, not an actual feminist group.

**Pull request closed without merging**